src/extensions/CommonJSModuleLoader.js

```diff
@@ -10,16 +10,15 @@ export default class CommonJSModuleLoader {
     }
 
     canLoadModule (extensionApi) {
-        return !!extensionApi.serviceDefinition.commonJS;
+        return !!extensionApi.serviceDefinition.module;
     }
 
     loadModule (extensionApi) {
 
-        const [modulePath, exportName] = extensionApi.serviceDefinition.commonJS.split('.');
-        const requiredModule = this.require(`./${modulePath}`);
-
-        return Promise.resolve(exportName ? requiredModule[exportName] : requiredModule);
+        const module = extensionApi.serviceDefinition.module;
```

Comment:
On further thought I actually disagree with this non configuration approach entirely and think it should be reverted. It missed the point of having the misdirection of the configuration. It will be impossible to add tooling support for IDEs and things with this approach. Looks at require.context docs for webpack, it's very powerful for only including the files you want. https://webpack.github.io/docs/context.html

Having a companion service that lets you use multiple require contexts, perhaps by prefix or something, is better.